apps/nx-mcp-e2e/src/tools.test.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  createInvokeMCPInspectorCLI,
+  e2eCwd,
+  newWorkspace,
+  simpleReactWorkspaceOptions,
+  uniq,
+} from '@nx-console/shared-e2e-utils';
+import { rmSync } from 'node:fs';
+import { join } from 'node:path';
+
+describe('tools', () => {
+  let invokeMCPInspectorCLI: (
```

Comment:
this could also be a type exported from the shared e2e utils package to make it more terse in the future